docs/excel/make-your-excel-add-in-backwards-compatible-with-com-add-in-or-xll.md

```diff
@@ -0,0 +1,112 @@
+
+To set the equivalent COM add-in, you specify its `ProgID`. Then Excel will create the mapping between your COM add-in UI and your web add-in UI.
+
+To set the equivalent XLL for your custom functions, specify the `FileName` of the XLL. When the user opens a workbook with functions from the XLL, it will convert the functions to compatible functions. Then the workbook will use the XLL when opened in Excel on desktop, and it will use custom functions from your web add-in when opened online or on macOS.
+
+The following example shows how to specify both a COM add-in and an XLL as equivalent. They are identified by their `ProgID` and `FileName` respectively.
+
+```xml
+<VersionOverrides>
+...
+<EquivalentAddins>  
+<EquivalentAddin>  
+       <ProgID>{progid}</ProgID>  
+       <Type>COM</Type>  
+  </EquivalentAddin>  
+  
+  <EquivalentAddin>  
+       <FileName>{filename}.xll</FileName>  
+       <Type>XLL</Type>  
+  </EquivalentAddin>  
+<EquivalentAddins>
+...
+</VersionOverrides>
+```
+
+> [!NOTE]
+> If an add-in declares its custom functions to be XLL compatible, changing the manifest at a later time could break a user’s workbook because it will change the file format.
+
+## How the compatibility behavior works for users
+
+When an equivalent COM add-in is specified, Excel will suppress your web add-in UI on Windows when the specified COM add-in is installed. This does not affect your web add-in UI on other platforms like online or macOS. The following scenarios describe what happens depending on how the user acquires the web add-in.
+
+### Office Store acquisition  
+
+If a user downloads the web add-in from the Office store Excel will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Centralized deployment
+
+If an admin deploys the web add-in to their tenant using centralized deployment, then the user needs to restart Excel before they will see any changes.
+
+After they restart Excel, Excel will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Document share
+
+If a user has the COM add-in installed, and then gets a shared document with the ebedded web add-in, then when they open the document:
+
+1. The user will see a trust prompt for the web add-in.
+2. If trusted, the web add-in will install.
+3. Excel hides the web add-in UI in the ribbon.
+
+### User uninstalls
+
+If a user uninstalls the COM add-in, Excel restores the web add-in UI on Windows if the web add-in was previously installed.
+
+Excel only hides the ribbon buttons and does not prevent installation. Therefore your web add-in will still appear in the following UI locations:
```

Comment:
I don't understand the point of this sentence (and the following two bullet points) here.